lib/appsignal.rb

```diff
@@ -267,15 +266,6 @@ def active?
       config && config.active? && extension_loaded?
     end
 
-    def is_ignored_error?(error)
-      Appsignal.config[:ignore_errors].include?(error.class.name)
-    end
-    alias :is_ignored_exception? :is_ignored_error?
-
-    def is_ignored_action?(action)
-      Appsignal.config[:ignore_actions].include?(action)
-    end
```

Comment:
I think it's likely people use these methods in custom integrations. We should deprecate them in 2.1 then.